sway/desktop/render.c

```diff
@@ -354,6 +354,11 @@ void render_box_shadow(struct sway_output *output, pixman_region32_t *output_dam
 	box.width += 2 * blur_sigma;
 	box.height += 2 * blur_sigma;
 
+	box.x += offset_x;
+	box.y += offset_y;
+	box.width -= offset_x;
+	box.height -= offset_y;
```

Comment:
Q: why do we change the height?

Reply:
I might suspect this to be the source of the negative offset issues

Reply:
When I don't adjust the with and height, the resulting area where the shadow will be drawn is only offset, not in contact with the inner box, creating a weird cutoff:
![image](https://github.com/WillPower3309/swayfx/assets/44938471/0df63f27-aa18-4ce7-b316-d76365bfbe18)

The same issue comes up when I move the box in a negative direction, and I haven't found why that happens or a way to fix that.
![a](https://github.com/WillPower3309/swayfx/assets/44938471/797d95ff-4ad4-4981-8753-644bee3b0f58)